oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -23,7 +23,18 @@
  */
 package oshi.software.os.linux;
 
-import static oshi.util.Memoizer.memoize;
+import org.slf4j.Logger;
```

Comment:
Uh oh. I bet our different IDEs have dueling import orderings... heh. I'll just do a mass formatting before merging this.

Reply:
Thanks i am not sure why my IDE is not formatting correctly. Sorry for the inconvenience caused.

Reply:
It's not a big deal. I should probably figure out the equivalent formatter settings for IntelliJ and save a file for those.  The formatting and "ordering imports" are separate things, though.